<ui version="4.0" >
 <class>pqOptionsFrame</class>
 <widget class="QDialog" name="pqOptionsFrame" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Options</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>4</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ApplyButton" >
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ResetButton" >
       <property name="text" >
        <string>&amp;Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseButton" >
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QStackedWidget" name="Stack" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="BlankPage" />
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QTreeView" name="PageNames" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="whatsThis" >
      <string>Select a category to show the options for that category.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QFrame" name="Separator" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CloseButton</tabstop>
  <tabstop>PageNames</tabstop>
  <tabstop>ApplyButton</tabstop>
  <tabstop>ResetButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
